### PR TITLE
Add Works and about me sections to portfolio.

### PR DESCRIPTION
**Pull request by ernestmusong**
I have added "works" (projects) and  "about me" sections to my portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+        <div class="cards-container">
+            <!-- project -->
+            <div class="project">
+                <div class="image-wrapper"></div>
+                <div class="project-footer">
+                    <div class="project-title-wrapper">
+                        <h4>Project tiltle goes here</h4>
+                    </div>
+                    <div class="project-buttons-wrapper">
+                        <button>ruby on rails</button>
```
Comment (Kossi-stack) [inline index.html:80]: - [ ] Kindly list all technologies in a list of "ul" and each of them  in "li"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+            <div class="card">
+                <div class="icon-wrapper">
+                    <div >
+                        <img class="icon" style="width: 100%; height: 100%; max-width: 100%;" src="images/icon-languages.png" alt="icon">
+                    </div>
+                </div>
+                <div class="card-title-wrap">
+                    <h4>Languages</h4>
+                </div>
+                <div class=" card-buttons-wrap">
```
Comment (Kossi-stack) [inline index.html:209]: - [ ] Kindly list all technologies in a list of "ul" and each of them in "li"

File: index.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+                <div class="icon-wrapper">
+                    <div>
+                        <img class="icon" style="width: 100%; height: 100%; max-width: 100%;" src="images/icon-skills.png" alt="icon">
+                    </div>
+                </div>
+                <div class="card-title-wrap">
+                    <h4>Languages</h4>
+                </div>
+                <div class=" card-buttons-wrap">
+                    <button>ruby on rails</button>
```
Comment (Kossi-stack) [inline index.html:244]: - [ ] Kindly list all category technologies in a list of "ul" and each of them in "li"

File: index.css [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+#bars {
+  height: 40px;
+  width: 40px;
+  display: grid;
+  place-items: center;
+}
+
+/* Headline */
+.about {
+  display: flex;
```
Comment (Kossi-stack) [inline index.css:57]: - [ ] Kindly use a display grid so it helps you align and balance with the design

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @ernestmusong,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.css (hunk 4 above)
Comment (ernestmusong) [reply index.css:61]: > * [ ]  Kindly use a display grid so it helps you align and balance with the design

Hi @Kossi-stack, thank you for your feedback. I have made all changes as indicated by you. However I think the headline section doesn't require a display grid as specified in the project's requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ernestmusong , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**